tests/test_hubspot_newfw_all_fields.py

```diff
@@ -0,0 +1,312 @@
+import unittest
+from tap_tester.base_suite_tests.all_fields_test import AllFieldsTest
+from tap_tester.logger import LOGGER
+from base_hubspot import HubspotBaseCase
+from client import TestClient
+
+def get_matching_actual_record_by_pk(expected_primary_key_dict, actual_records):
+    ret_records = []
+    can_save = True
+    for record in actual_records:
+        for key, value in expected_primary_key_dict.items():
+            actual_value = record[key]
+            if actual_value != value:
```

Comment:
Would like to run this and see in debugger.

Reply:
Let me set up a time for this and to discuss the above comment